meshroom/ui/graph.py

```diff
@@ -183,10 +183,12 @@ def watchedStatusFiles(self):
             return self.statusFiles, self.monitorableChunks
         elif self.filePollerRefresh is PollerRefreshStatus.MINIMAL_ENABLED.value:
             for c in self.monitorableChunks:
-                # When a chunk's status is ERROR, it may be externally re-submitted and it should thus still be monitored
-                if c._status.status is Status.SUBMITTED or c._status.status is Status.RUNNING or c._status.status is Status.ERROR:
-                    files.append(c.statusFile)
-                    chunks.append(c)
+                # Only chunks that are run externally should be monitored; when run locally, status changes are already notified
+                if c._status.execMode is ExecMode.EXTERN:
```

Comment:
That solves the conflict issue for monitoring chunks being computed in the current meshroom instance :+1:.

But could we even use `c.isExtern()` here? 
To handle both remote computation and local computation happening within another Meshroom instance (the 2 cases were monitoring via polling makes sense)?

Reply:
You're totally right, I'll make that change!

Reply:
Done!